### PR TITLE
Changed default hostname to openhab

### LINKED ISSUE #177: [Proposal] Change default hostname to "openhab"
I'm aware that there's an option to easily change the hostname by yourself, although it might be cool for the newcomers to just type `openhab:8080` (12 chars) instead `openhabianpi:8080` (17 chars) 😉 

My personal preference is `home`. Combined with my [local-alias](https://kubawolanin.github.io/local-alias/) it's pretty handy:

### PR DESCRIPTION
**Pull request by kubawolanin**
Fix #177 

- I've deliberately used lowercase everywhere - I doubt users will bother changing case to "openHAB" when they type the URL in the address bar
- Cleaned up the Amanda docs as well, so their example correspond with the default hostname. Pinging @mstormi :-)

@ThomDietrich FYI I left two separate config files for Pine64 and Pi even though they are the same for now. Who knows if the initial configuration of them changes in the future, so I'd rather keep that separated.

Cheers!

### CODE REVIEW

File: docs/openhabian.md [hunk 1]
```diff
@@ -313,11 +313,11 @@ We will re-add the functionality as soon as the underlying issue is resolved.
...
 
-After the installation of openHABian was successful, you should be able to access the openHAB dashboard:
+After the installation of openHABian was successful, you should be able to access the openHAB dashboard
+with the following URLs:
 
-- Raspberry Pi image setup: [http://openhabianpi:8080](http://openhabianpi:8080)
-- Pine A64 image setup: [http://openhabianpine64:8080](http://openhabianpine64:8080)
-- In any case: [http://your-device-hostname:8080](http://your-device-hostname:8080) or [http://192.168.0.2:8080](http://192.168.0.2:8080) (replace name/IP)
+* [http://openhab:8080](http://openhab:8080)
+* e.g. [http://192.168.0.2:8080](http://192.168.0.2:8080) (replace IP accordingly)
```
Comment (ThomDietrich) [inline docs/openhabian.md]: why a line break in the sentence above?

In the list I would suggest to prepend something like "hostname or DNS name:" and "IP:"

File: functions/helpers.sh [hunk 2]
```diff
@@ -62,7 +62,7 @@ is_pithreeplus() {
 }
 is_pi() {
   # needed for raspbian-ua-netinst chroot env #TODO can be removed?
-  if [ "$hostname" == "openHABianPi" ] || [ "$boot_volume_label" == "openHABian" ]; then return 0; fi
+  if [ "$boot_volume_label" == "openHABian" ]; then return 0; fi
```
Comment (ThomDietrich) [inline functions/helpers.sh]: Those two lines can be deleted

**Comment by ThomDietrich** [comment]
Looks good. Thanks!
Regarding lowercase: Not sure what you mean. Isn't a URL not case-sensitive!?

**Comment by mstormi** [comment]
URLs *are* case sensitive.
But hostnames and the hostname part of a URL are not, so I wouldn't expect any harm from Kuba's changes.

**Comment by ThomDietrich** [comment]
This... I live in the IT world for almost 20 years ans did not know this 🙈

**Comment by kubawolanin** [comment]
Hey Thomas, I've addressed your comments 👨‍💻 
Thank you for the review!

**Comment by ThomDietrich** [comment]
I was about to merge when I realized that this change will (a) only effect newly setup installations with a to be releases image, however (b) change details in the documentation article.

Therefore I'll merge the PR shortly before releasing a new image

**Comment by EliasGabrielsson** [comment]
@kubawolanin Hi Kuba,

Could you rebase this on master and make the hostname changes in:
`build--image/openhabian.pine64-xenial.conf` and `build--image/openhabian.pine64-raspbian.conf` ?

Thanks

**Comment by kubawolanin** [comment]
@EliasGabrielsson sure thing, consider it done! 😎

**Comment by vromero** [comment]
Could you share an status update around this? I was about to work on a  PR around the same, but also I see this is months old, so not sure what is going on.

**Comment by kubawolanin** [comment]
> Could you share an status update around this? I was about to work on a PR around the same, but also I see this is months old, so not sure what is going on.

Of course. There were some minor conflicts in the Markdown docs which I have now resolved.
This PR is ready from my side.

Cheers!

**Comment by vromero** [comment]
Lovely, just FWIW pinging maintainers (woudn't typically take the liberty, but given that it's months since the last update I guess I might): @ThomDietrich @mstormi @EliasGabrielsson

**Comment by EliasGabrielsson** [comment]
Hi @vromero. 
You are correct that things take time, can just say sorry for that. 
The reason this ain't got merged yet is we need to review all documentation while doing so.
I have had a goal of making a new release when all v1.5 issues are complete and then also merge this.

It basically boils down to the constrain amount of time we as maintainers have.

**Comment by vromero** [comment]
That's completely understandable @EliasGabrielsson. I appreciate the heads up.

Thank you for the time and effort you guys put in OpenHab. It's a superb project.

**Comment by EliasGabrielsson** [comment]
@kubawolanin Thanks for your work with this.
I superseeded this PR by https://github.com/openhab/openhabian/pull/630.

**Pull request closed without merging**